ckanext/scheming/templates/scheming/snippets/form_field.html

```diff
@@ -53,8 +35,12 @@
   {# A complex field with sub-fields, which can optionally be repeatable. #}
   {% set flat = h.scheming_flatten_subfield(field, data) %}
   {% set flaterr = h.scheming_flatten_subfield(field, errors) %}
-  <fieldset name="scheming-repeating-subfields" class="scheming-fieldset" data-module="scheming-repeating-subfields">
-    {% set alert_warning = h.scheming_language_text(field.form_alert_warning) %}
+  {% block removal_text %}
+    {% set locale = h.dump_json({'content': _('These fields have been removed, click save below to update the record with your changes.') }) %}
+
+<fieldset name="scheming-repeating-subfields" class="scheming-fieldset" data-module="scheming-repeating-subfields" data-module-i18n="{{ locale }}">
+  {% endblock %}
```

Comment:
little weird to end a block at a different html tag level, is this just to allow the locale variable to be visible?

Reply:
Yes, I just didn't want to include the entire fieldset element but it doesn't make much of a difference where it goes. Should I include the entire element or just the locale variable?

Reply:
I see there's an "options" section in the module definition and the string is repeated in the `set` statement here. What is the purpose of the options section? Apologies for my limited JS knowledge, if it's easier to explain on a call we could do that instead.

Reply:
The string in the module options is the default text, which can be changed through the locale variable. The locale var is included in the block to allow for a custom message and in this case is just a place holder